.travis.yml

```diff
@@ -61,7 +61,7 @@ install:
   - rm -rf "$HOME/.m2/repository/org/apache/beam"
 
 script:
-  - travis_retry mvn --batch-mode --update-snapshots --no-snapshot-updates $MAVEN_OVERRIDE install && travis_retry bash -ex .travis/test_wordcount.sh
+  - travis_retry mvn --batch-mode --update-snapshots $MAVEN_OVERRIDE install && travis_retry bash -ex .travis/test_wordcount.sh
```

Comment:
yes, my plan was to temporarily remove this in a separate commit which you can exclude with cherry-pick but it didn't work either.